brayns/common/parameters/SimulationParameters.cpp

```diff
@@ -81,4 +81,19 @@ void SimulationParameters::build(ArgvBuilder &builder)
 {
     (void)builder;
 }
+
+void SimulationParameters::resetModified()
+{
+    _flag = false;
+}
+
+ModifiedFlag &SimulationParameters::getModifiedFlag() noexcept
+{
+    return _flag;
+}
+
+const ModifiedFlag &SimulationParameters::getModifiedFlag() const noexcept
+{
+    return _flag;
+}
```

Comment:
Like for the transfer function, can't we just have isModified() and resetModified() as public method ?

(So the flag is hidden as an implementation detail)